src/GZCTF/Program.cs

```diff
@@ -292,15 +294,15 @@
 
 app.UseForwardedHeaders();
 
-if (app.Environment.IsDevelopment())
+if (!app.Environment.IsDevelopment())
```

Comment:
Did you forget to flip it after testing?

Reply:
Sorry for my carelessness